CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # credential-handler-polyfill ChangeLog
 
+## 1.1.0 - 2019-xx-xx
```

Comment:
This should probably be 2.0.0 due to the name change. In case anyone was using the amd output with the previous name. Would need to change demo link too.

Reply:
And readme link.

Reply:
@davidlehn Done, thanks!